Packs/CortexXDR/Integrations/CortexXDRIR/CortexXDRIR_test.py

```diff
@@ -248,8 +307,7 @@ def test_get_starred_incident_list_with_limit(self, mocker):
             'incident_id_list': '1 day',
             'starred': True,
             'limit': 1,
-            'starred_incidents_fetch_window': '3 days',
-            'integration_context_brand': 'PaloAltoNetworksXDR'
+            'starred_incidents_fetch_window': '3 days'
```

Comment:
Why was it removed?

Reply:
I reverted the changes (used checkout from master) and rewritten the tests